src/cmd/build.js

```diff
@@ -97,9 +102,19 @@ export async function getDefaultLocalizedName(
 export type PackageCreatorFn =
     (params: PackageCreatorParams) => Promise<ExtensionBuildResult>;
 
-async function defaultPackageCreator({
-  manifestData, sourceDir, fileFilter, artifactsDir, showReadyMessage,
-}: PackageCreatorParams): Promise<ExtensionBuildResult> {
+export async function defaultPackageCreator(
+  {
+    manifestData,
+    sourceDir,
+    fileFilter,
+    artifactsDir,
+    overwriteDest,
+    showReadyMessage,
+  }: PackageCreatorParams,
+  {
+    eventToPromise = defaultEventToPromise,
+  }:PackageCreatorOptions = {}
```

Comment:
This line has a lint warning (a space is missing between `:` and `PackageCreatorOptions`):

```
$ npm run lint
> grunt lint

Running "lint" task

Running "eslint:js" (eslint) task

/.../web-ext/src/cmd/build.js
  114:3  warning  There must be a space after "{
    eventToPromise = defaultEventToPromise,
  }:PackageCreatorOptions" parameter type annotation colon  flowtype/space-after-type-colon

✖ 1 problem (0 errors, 1 warning)
```